modules/gitops-storing-and-retrieving-argo-cd-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-storing-and-retrieving-argo-cd-logs_{context}"]
+= Storing and retrieving Argo CD logs
+
+You can use the Kibana dashboard to store and retrieve Argo CD logs.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the {rh-app-icon} menu -> *Observability* -> *Logging*. The Kibana dashboard is opened in a new window.
+
+. Create an index pattern.
+
+.. To display all the indeces, define the index pattern as "***" . Click *Next step* to continue.
+
+.. Select *@timestamp* for _Time Filter field name_.
+
+.. Click *Create index pattern* and wait until the index pattern is created.
+
+. In the left navigation panel of the Kibana dashboard, click the *Discover* tab.
+
+. Create a filter to retrieve logs for Argo CD.
+
+.. Click *Add a filter +*.
+
+.. Provide `kubernetes.namespace_name` as the field. Select the "*is*" operator and enter `openshift-gitops` as a value.
++
+This filter retrieves logs for all the pods in the `openshift-gitops` namespace.
+
+.. Click *Save* to create the filter.
+
+. You can add additional filters to narrow the search. For example, to retrieve logs for a particular pod, create another filter with `kubernetes.pod_name` as the field.
```

Comment:
Nitpick: I'd recommending adding `can` to the example to make it easier to read.
```suggestion
. You can add additional filters to narrow the search. For example, to retrieve logs for a particular pod, you can create another filter with `kubernetes.pod_name` as the field.
```